robotoff/insights/ocr/packager_code.py

```diff
@@ -29,13 +28,24 @@ def process_fr_emb_match(match) -> str:
     return "EMB {}{}".format(city_code, company_code).upper()
 
 
+def process_fsc_match(match) -> str:
+    fsc_code = match.group(1)
+    return "FSC-{}".format(fsc_code)
+
+
 PACKAGER_CODE: Dict[str, OCRRegex] = {
     "fr_emb": OCRRegex(
         re.compile(r"emb ?(\d ?\d ?\d ?\d ?\d) ?([a-z])?(?![a-z0-9])"),
         field=OCRField.text_annotations,
         lowercase=True,
         processing_func=process_fr_emb_match,
     ),
+    "fsc": OCRRegex(
+        re.compile(r"FSC.? (C\d{6})"),
+        field=OCRField.text_annotations,
```

Comment:
filed=OCRField.full_text is better here, we should use it in other packager code regexes as well.

Reply:
@raphael0202  I tried with full_text and with full_text_contiguous and it does not work for code 20126803. Only text_annotations works for this code.

Reply:
I see, indeed full_text is not always present. My bad.